src/queries/queries.js

```diff
@@ -0,0 +1,92 @@
+import { gql } from 'react-apollo'
```

Comment:
How shall we sort these queries? Did you sort them according to categories? As I see you didn't sort them in alphabetical order. Maybe we should name them as their name express the component in which it is used.